internal/config/validation.go

```diff
@@ -19,19 +19,6 @@ import (
 	"github.com/go-playground/validator/v10"
 )
 
-func validateImage(fl validator.FieldLevel) bool {
-	image := fl.Field().String()
-	switch {
-	case image == "":
-		return false
-	case image == "..":
```

Comment:
Shouldn't this be a `strings.Contains` as well?

Reply:
So there's no need for this function anymore?

Reply:
Yes I can remove it!